api/src/main/resources/public/api/v0/schema/jobspec_v0.schema.json

```diff
@@ -3,7 +3,7 @@
   "definitions": {
     "pathType": {
       "type": "string",
-      "pattern": "^([a-z0-9]([a-z0-9-]*[a-z0-9]+)*)([.][a-z0-9]([a-z0-9-]*[a-z0-9]+)*)*$",
```

Comment:
My hypothesis is that someone has thrown an id at the validator which caused Catastrophic backtracking http://www.regular-expressions.info/catastrophic.html